### PR TITLE
media-libs/libffado: bump to 2.4.2

### PR DESCRIPTION
**Pull request by internethering**
Signed-off-by: Richard hering <richard@internethering.de>
Package-Manager: Portage-2.3.78, Repoman-2.3.17

### CODE REVIEW

File: media-libs/libffado/libffado-2.4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (juippis) [inline media-libs/libffado/libffado-2.4.2.ebuild]: Could you try with 7? I think all the eclasses should support it.

File: media-libs/libffado/libffado-2.4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	sys-apps/dbus
+	sys-libs/libavc1394[${MULTILIB_USEDEP}]
+	sys-libs/libraw1394[${MULTILIB_USEDEP}]
+	qt5? (
+		dev-python/dbus-python[${PYTHON_USEDEP}]
+		dev-python/PyQt5[dbus,${PYTHON_USEDEP}]
+		x11-misc/xdg-utils
+	)"
+DEPEND="${COMMON_DEPEND}
+	virtual/pkgconfig"
```
Comment (juippis) [inline media-libs/libffado/libffado-2.4.2.ebuild]: (With EAPi-7 this is a BDEPEND)

**Comment by marcan** [comment]
False alarm, looks like the ebuild in this PR works fine. Looks fine modulo what @juippis mentioned.

**Comment by juippis** [comment]
Ping, could you address my comments?

**Pull request closed without merging**